### ISSUE TITLE
Crash sharing downloaded file to nextcloud

### ISSUE THREAD

**Issue by arlinsandbulte** [nextcloud/android#11578]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Download a file.
Share file to nextcloud.
Crash happens.

### Expected behaviour

Add downloaded file to nextcloud.

### Cause of error
```java
Exception in thread "Thread-19" java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.datamodel.OCFile.getRemotePath()' on a null object reference
    at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:230)
    at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:399)
    at java.lang.Thread.run(Thread.java:1012)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30240290`
* Build flavor: `gplay`

### Device information
* Brand: `google`
* Device: `sargo`
* Model: `Pixel 3a`
* Id: `SP2A.220505.008`
* Product: `sargo`

### Firmware
* SDK: `32`
* Release: `12`
* Incremental: `8782922`

### Android version

12

### Device brand and model

Pixel 3a

### Stock or custom OS?

Stock

### Nextcloud android app version

3.24.2

### Nextcloud server version

25.0.6

### Using a reverse proxy?

No

**Comment by mainboarder** [comment]
https://github.com/nextcloud/android/issues/11207 seems to be the first occurence as bug

**Comment by joshtrichards** [comment]
Fixed in #11410 pending the next release.
